code/controllers/subsystem/vote.dm

```diff
@@ -180,6 +180,8 @@ SUBSYSTEM_DEF(vote)
 	return .
 
 /datum/controller/subsystem/vote/proc/submit_vote(vote)
+	// Voting where vote power is equal for all
+	var/list/everyone_is_equal = list("custom")
```

Comment:
I dunno why this is IN the proc rather than on the subsystem, but at least, like, `/static/`?